src/tests/control/test_orders.py

```diff
@@ -105,6 +105,18 @@ def test_order_set_contact(client, env):
 
 
 @pytest.mark.django_db
+def test_order_set_locale(client, env):
+    q = Quota.objects.create(event=env[0], size=0)
+    q.items.add(env[3])
+    client.login(email='dummy@dummy.dummy', password='dummy')
+    client.post('/control/event/dummy/dummy/orders/FOO/locale', {
+        'locale': 'de'
+    })
+    o = Order.objects.get(id=env[2].id)
+    assert o.locale == 'de'
```

Comment:
Yay, tests! A second one showing the result of `POST`ing an illegal locale would be nice, too.